docs/Concepts/Plugins.md

```diff
@@ -0,0 +1,46 @@
+description: Plugins
+<!--- END of page meta data -->
+
+# Plugins
+
+You can extend Hyperledger Besu functionality by building Java plugins. The
+Plugin API allows you to take data from any Besu network, public or
+permissioned, and feed it into an application or system. 
```

Comment:
The Plugin API allows you to -> Use the Plugin API 

Suggested edit to make active

Reply:
Done.